src/resources/views/auth/choice.blade.php

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="ja">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>POSSE課題</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+
+    <p>{{$questions[$question_id-1]->id}}</p>
+    <form action="/admin/choice/{{$big_question_id}}/{{$question_id}}" method="POST">
```

Comment:
route関数を利用した方が良いかと思います
```suggestion
    <form action="{{ route(命名したやつ) }}" method="POST">
```